index.js

```diff
@@ -111,6 +111,28 @@ Babel.prototype.optionsHash = function() {
       hash[key] = (typeof value === 'function') ? (value + '') : value;
     }
 
+    if (options.plugins) {
+      hash.plugins = [];
```

Comment:
> What this PR adds, would be a complement to the hash-fpr-dep approach, not the other way around.

How is this a useful point? Both things are required, I have already agreed with that.